apps/remix-ide/src/app.js

```diff
@@ -159,6 +185,7 @@ class AppComponent {
     const contextualListener = new ContextualListener({ editor })
 
     self.engine.register([
```

Comment:
you also need to to 
`await self.appManager.activatePlugin(['app'])` (line 293)

Reply:
doing this doesn't remix and shows errors in browser console